mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -345,10 +390,12 @@ CLASS aij net/minecraft/entity/Entity
 	METHOD bQ getServer ()Lnet/minecraft/server/MinecraftServer;
 	METHOD bR isImmuneToExplosion ()Z
 	METHOD bS entityDataRequiresOperator ()Z
+	METHOD bT teleportRequested ()Z
 	METHOD bU getPrimaryPassenger ()Laij;
 	METHOD bV getPassengerList ()Ljava/util/List;
 	METHOD bW getPassengersDeep ()Ljava/util/Collection;
-	METHOD bY getTopmostVehicle ()Laij;
+	METHOD bX hasPlayerJockey ()Z
```

Comment:
IMO Jockey implies that the player is controlling this entity. In `PlayerManager#remove`, we can see the following code:
```java
if (entity_1.method_5817()) {
    LOGGER.debug("Removing player mount");
```
So why not call it `isPlayerMount` ?

Reply:
isPlayerMount or isPlayerMounted?

Reply:
`hasPlayerMount`?

Reply:
`hasPlayerMount`

Reply:
What ? No, this returns true if the entity is mounted by a player.